oauthdoctor/diag/config.go

```diff
@@ -130,7 +130,7 @@ var Languages = map[string]Config{
 		Comment: Comment{
 			LeftMeta: "#",
 		},
-		Separator:   ":",
+		Separator: ":",
```

Comment:
Do you use gofmt before submitting?

Reply:
I was switching between vim and Visual Studio Code, and the formatting tool setup is different. That's why it's shifting spaces around.